index.js

```diff
@@ -10,18 +10,31 @@ const contrib = require('blessed-contrib');
 const moment = require('moment');
 const program = require('commander');
 
-program.version('0.0.1');
+program.version('0.1.0');
 program
   .option('-n, --stack-name <stackName>', 'AWS stack name')
   .option('-r, --region <region>', 'AWS region')
   .option('-t, --start-time <startTime>', 'when to start from')
-  .option('-p, --period <period>', 'precision of graphs, in seconds')
+  .option('-i, --interval <interval>', 'interval of graphs, in seconds')
+  .option('-p, --profile <profile>', 'aws profile name to use')
   .parse(process.argv);
 
 const screen = blessed.screen({ smartCSR: true });
-const cloudformation = new AWS.CloudFormation({ region: program.region });
-const cloudwatch = new AWS.CloudWatch({ region: program.region });
-const cloudwatchLogs = new AWS.CloudWatchLogs({ region: program.region });
+const profile = program.profile || 'default';
+console.log(program.profile);
+const credentials = new AWS.SharedIniFileCredentials({ profile });
+const cloudformation = new AWS.CloudFormation({
```

Comment:
`AWS.config.credentials = credentials;`

Reply:
See https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/loading-node-credentials-shared.html